creating_images/guidelines.adoc

```diff
@@ -297,47 +297,69 @@ to allow users in the root group to access them in the built image:
 
 ----
 RUN chgrp -R 0 /some/directory && \
-    chmod -R g=u /some/directory
+    chmod -R g+rwX /some/directory
 ----
 
 Because the container user is always a member of the root group, the container
-user can read and write these files. The root group does not have any special
-permissions (unlike the root user) so there are no security concerns with this
-arrangement. In addition, the processes running in the container must not listen
-on privileged ports (ports below 1024), since they are not running as a
+user can read and write these files.
+
+[WARNING]
+====
+Care must be taken when altering the directories and file permissions of
+sensitive areas of a container (no different than to a normal system).
+
+If applied to sensitive areas, such as /etc/passwd, this can allow the
```

Comment:
s//etc/passwd/`/etc/passwd` 

Applies to each reference.

Reply:
or *_/etc/passwd_* for consistency.

Reply:
agreed